cmatrix.c

```diff
@@ -630,6 +635,11 @@ if (console) {
                         while (i <= LINES && (matrix[i][j].val != ' ' &&
                                matrix[i][j].val != -1)) {
                             matrix[i][j].is_head = false;
+                            if(changes) {
+                                int dice = rand() % 8;
+                                if(dice == 0)
```

Comment:
Instead of assigning a variable, since we are are not reusing dice, can we remove the variable and make this if into something like: `rand()%8 == 0`

Or maybe perhaps store the number from `rand()` in dice, make the if into `dice%8 == 0` and then reuse dice in the next line?

If you think these changes are minor and not worthy, then let me know, and we can merge it this way (as it is now).